cumulusci/tasks/github.py

```diff
@@ -207,3 +210,51 @@ def _run_task(self):
                 )
 
                 self.logger.info('Merge conflict on branch {}: created pull request #{}'.format(branch.name, pull.number))
+
+
+class CommitApexDocs(BaseGithubTask):
+
+    task_options = {
```

Comment:
Agreed for `branch` value but for `dir_local` this is the output of the apex docs command, not sure we'd want that set by default. If so, the value should also be used in generation of ApexDocs.